dataland-frontend/tests/component/components/resources/frameworkDataSearch/gdvVoeb/GdvVoebPanel.cy.ts

```diff
@@ -0,0 +1,153 @@
+import { type FixtureData, getPreparedFixture } from "@sharedUtils/Fixtures";
+import { type CompanyAssociatedDataGdvData, DataTypeEnum, type GdvData } from "@clients/backend";
+
+import { mountMLDTFrameworkPanelFromFakeFixture } from "@ct/testUtils/MultiLayerDataTableComponentTestUtils";
+import { type MLDTConfig } from "@/components/resources/dataTable/MultiLayerDataTableConfiguration";
+
+import { type AvailableMLDTDisplayObjectTypes } from "@/components/resources/dataTable/MultiLayerDataTableCellDisplayer";
+import { formatGdvYearlyDecimalTimeseriesDataForTable } from "@/components/resources/dataTable/conversion/gdv/GdvYearlyDecimalTimeseriesDataGetterFactory";
+import {
+  formatStringForDatatable
+} from "@/components/resources/dataTable/conversion/PlainStringValueGetterFactory";
+import {
+  formatListOfBaseDataPoint
+} from "@/components/resources/dataTable/conversion/gdv/GdvListOfBaseDataPointGetterFactory";
+import {formatNumberToReadableFormat} from "../../../../../../src/utils/Formatter";
+
+const configForGdvVoebPanelWithOneRollingWindow: MLDTConfig<GdvData> = [
+  {
+    type: "cell",
+    label: "Überwachung der Einkommensungleichheit",
+    explanation:
+        "Bitte geben Sie das unbereinigte geschlechtsspezifische Lohngefälle, das Einkommensungleichheitsverhältnis, sowie das CEO-Einkommensungleichheitsverhältnis für die letzten drei Jahre an.",
+    shouldDisplay: (): boolean => true,
+    valueGetter: (dataset: GdvData): AvailableMLDTDisplayObjectTypes =>
+        formatGdvYearlyDecimalTimeseriesDataForTable(
+            dataset.soziales?.einkommensgleichheit?.ueberwachungDerEinkommensungleichheit,
+            {
+              geschlechtsspezifischesLohngefaelle: {
+                label: "Geschlechtsspezifisches Lohngefälle",
+                unitSuffix: "%",
+              },
+              einkommensungleichheitsverhaeltnis: { label: "Einkommensungleichheitsverhältnis", unitSuffix: "%" },
+              ceoEinkommenungleichheitsverhaeltnis: {
+                label: "CEO-Einkommensungleichheitsverhältnis",
+                unitSuffix: "%",
+              },
+            },
+            "\u00DCberwachung der Einkommensungleichheit",
+        ),
+  },
+];
+
+const configForGdvVoebPanelWithOneStringComponent: MLDTConfig<GdvData> = [
+  {
+    type: "cell",
+    label: "Wirtschaftsprüfer",
+
+    shouldDisplay: (): boolean => true,
+    valueGetter: (dataset: GdvData): AvailableMLDTDisplayObjectTypes =>
+        formatStringForDatatable(dataset.unternehmensfuehrungGovernance?.wirtschaftspruefer),
+  },
+];
+
+const configForGdvVoebPanelWithOneListForBaseDataPointComponent: MLDTConfig<GdvData> = [
+  {
+    type: "cell",
+    label: "Aktuelle Berichte",
+    explanation: "Aktuelle Nachhaltigkeits- oder ESG-Berichte",
+    shouldDisplay: (): boolean => true,
+    valueGetter: (dataset: GdvData): AvailableMLDTDisplayObjectTypes => {
+      return formatListOfBaseDataPoint(
+          "Aktuelle Berichte",
```

Comment:
Here the same string is used as in the label field in line 57. Why do we need those?

Reply:
@marcelrasbachdfine  can you pls answer this question?  I don't know if there are some thoughts behind this.

Reply:
Test will be redesigned by Emnauel to adress several issues. Once that is done both comments at this file can be closed

Reply:
Looks fine now